src/Uno.UI/UI/Xaml/Shapes/Rectangle.cs

```diff
@@ -22,7 +22,7 @@ protected override Size MeasureOverride(Size availableSize)
 			typeof(Rectangle),
 			new FrameworkPropertyMetadata(
 				defaultValue: 0.0,
-				options: FrameworkPropertyMetadataOptions.AffectsArrange
+				options: FrameworkPropertyMetadataOptions.AffectsMeasure
```

Comment:
Hmm.. I don't think this should be affecting measure. But if that's what WinUI is doing let's follow that. I wouldn't expect this property to be changing dynamically that much in real scenarios, so doing the potentially unnecessary measure shouldn't be a problem.

Reply:
It does according to this:

![image](https://github.com/unoplatform/uno/assets/1075116/2a6ef524-317e-46a0-bdc8-b5128970beb1)

It is interesting though, maybe they do the calculations for it in measure